src/components/index.ts

```diff
@@ -0,0 +1,6 @@
+export {default as Loader} from './Loader';
```

Comment:
To avoid this patter, when using TS it is suggested to avoid using the default export. If you use named export you then can just do: ```export * form './Loader'```

Reply:
I used this approach: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#re-exporting_aggregating